docs/docs/configuration/client.rst

```diff
@@ -100,6 +101,16 @@ override the API location:
 data-isso-css
 -------------
 
+Set URL from which to fetch ``isso.css``, e.g. from a CDN.
+Defaults to fetching from the API endpoint.
+
+.. code-block:: html
+
+    <script src="..." data-isso-css-url="/path/to/isso.css"></script>
+
+data-isso-css-url
```

Comment:
I guess this block was put at the wrong place. The text above this heading is for the attribute data-isso-css-url. Mind to move it below?

Reply:
Thanks for catching this! Fixed.